bitnami/postgresql-repmgr/17/debian-12/rootfs/opt/bitnami/scripts/libpostgresql.sh

```diff
@@ -747,6 +747,46 @@ postgresql_initialize() {
     postgresql_stop
 }
 
+########################
+# Run custom start scripts
+# Globals:
+#   POSTGRESQL_*
+# Arguments:
+#   None
+# Returns:
+#   None
+#########################
+postgresql_custom_start_scripts() {
```

Comment:
This is almost the same code as in `postgresql_custom_init_scripts`: https://github.com/bitnami/containers/blob/da31f8d62f4dee863feaf393d6db70d09e6a5394/bitnami/postgresql/17/debian-12/rootfs/opt/bitnami/scripts/libpostgresql.sh#L784

To avoid duplication, could you follow the same approach we've used in `mariadb`? https://github.com/bitnami/containers/blob/da31f8d62f4dee863feaf393d6db70d09e6a5394/bitnami/mariadb/11.4/debian-12/rootfs/opt/bitnami/scripts/libmariadb.sh#L381-L390